AppCreationScripts/Apps.json

```diff
@@ -11,11 +11,11 @@
       "x-ms-version": "2.0",
       "replyUrlsWithType": [
         {
-          "url": "https://localhost:8080/msal4jsample/secure/aad",
+          "url": "https://localhost:8443/msal4jsample/secure/aad",
```

Comment:
I think if we change the url, we would also have to update the quickstart. I added @navyasric to the reviewers to make sure we don't break the quickstart experience